### DISCUSSION TITLE
Add themed helper classes for text decoration color

### DISCUSSION THREAD

**Discussion by mosquitobird11** [saadeghi/daisyui#1926] [Ideas / feature requests]
https://tailwindcss.com/docs/text-decoration-color 
adding `decoration-primary` `decoration-secondary` `decoration-accent` etc. would be a nice addition!

**Comment by saadeghi** [comment]
It already works. Any Tailwind CSS class name that uses Tailwind CSS color names also works with daisyUI color names so you can use `underline decoration-primary` and it works.
